ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParser.scala

```diff
@@ -49,16 +49,30 @@ object WorkflowConfigParser {
     }
 
     implicit val partyFilterDecoder: Decoder[StreamConfig.PartyFilter] =
-      Decoder.forProduct2(
-        "party",
-        "templates",
-      )(StreamConfig.PartyFilter.apply)
+      (c: HCursor) => {
+        for {
+          party <- c.downField("party").as[String]
+          templates <- c.downField("templates").as[List[String]]
```

Comment:
Will we want to always specify `templates`?

Reply:
good point, it might make sense to make it optional, will update it